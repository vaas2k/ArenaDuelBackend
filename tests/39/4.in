10
7 8
10 10
2 2
8 9
10 10
5 6
9 10
1 4
10 10
4 8
