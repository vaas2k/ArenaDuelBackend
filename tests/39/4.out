4
1 4 2 3 3 1 1 1 2 2 
